### PR TITLE
Apply overrides in FWTW

### PR DESCRIPTION
**Pull request by liuly12**
`initial_storage` is currently not enabled

### CODE REVIEW

File: tests/test_wtw.py [hunk 1]
```diff
@@ -113,6 +113,26 @@ def test_wwtw_overrides(self):
...
+                              })
+        self.assertEqual(fwtw.treatment_throughput_capacity, 20)
+        self.assertEqual(fwtw.process_parameters['phosphate']['constant'], 0.01)
+        self.assertEqual(fwtw.process_parameters['volume']['constant'], vol)
+        self.assertEqual(fwtw.service_reservoir_storage_capacity, 100)
+        self.assertEqual(fwtw.service_reservoir_tank.capacity, 100)
+        self.assertEqual(fwtw.service_reservoir_storage_area, 34.7)
+        self.assertEqual(fwtw.service_reservoir_tank.area, 34.7)
+        self.assertEqual(fwtw.service_reservoir_storage_elevation, 68.2)
+        self.assertEqual(fwtw.service_reservoir_tank.datum, 68.2)
```
Comment (barneydobson) [inline tests/test_wtw.py:135]: Maybe just to check it doesn't delete other bits
```suggestion
        self.assertEqual(fwtw.service_reservoir_tank.datum, 68.2)
        self.assertEqual(fwtw.process_parameters['nitrate']['constant'], 0.01)
```

File: tests/test_wtw.py [hunk 2]
```diff
@@ -113,6 +113,26 @@ def test_wwtw_overrides(self):
         self.assertEqual(wwtw.process_parameters['phosphate']['constant'], 0.01)
         self.assertEqual(wwtw.process_parameters['volume']['constant'], vol)
         self.assertEqual(wwtw.stormwater_storage_capacity, 100)
+    
+    def test_fwtw_overrides(self):
+        fwtw = FWTW(name='')
+        vol = fwtw.process_parameters['volume']['constant']
+        fwtw.apply_overrides({'treatment_throughput_capacity' : 20,
+                              'process_parameters' : {'phosphate' : 
+                                                      {'constant' : 0.01}},
```
Comment (barneydobson) [inline tests/test_wtw.py:122]: Maybe use something that isn't the default value ;)

```suggestion
                                                      {'constant' : 0.02}},
```

File: tests/test_wtw.py [hunk 3]
```diff
@@ -113,6 +113,26 @@ def test_wwtw_overrides(self):
...
+                              'service_reservoir_storage_capacity': 100,
+                              'service_reservoir_storage_area': 34.7,
+                              'service_reservoir_storage_elevation': 68.2
+                              })
+        self.assertEqual(fwtw.treatment_throughput_capacity, 20)
+        self.assertEqual(fwtw.process_parameters['phosphate']['constant'], 0.01)
```
Comment (barneydobson) [inline tests/test_wtw.py:128]: RE above
```suggestion
        self.assertEqual(fwtw.process_parameters['phosphate']['constant'], 0.02)
```

**Review by barneydobson — APPROVED** [review]
Some basic suggestions for tests - otherwise happy

**Review by barneydobson — APPROVED** [review]

**Review by dalonsoa — APPROVED** [review]
LGTM, now that tests and QA pass :)

**Pull request merged**